### DISCUSSION TITLE
Problem with excludes yaml

### DISCUSSION THREAD

**Discussion by artesburger** [dadoonet/fscrawler#1524] [Q&A]
Bonjour, 
J'index un répertoire mapper du nom de R:/, je préciser dans mon yaml du job, d'exclude un dossier du nom de #recycle.
Mais à chaque fois le dossier s'index. Je vous mets quelques logs :

14:41:53,321 DEBUG [f.p.e.c.f.f.FsCrawlerUtil] directory = [true], filename = [#recycle], includes = [null], excludes = [[#recyle]]
14:41:53,321 DEBUG [f.p.e.c.f.f.FsCrawlerUtil] filename = [#recycle], excludes = [[#recyle]]
14:41:53,321 DEBUG [f.p.e.c.f.f.FsCrawlerUtil] filename = [#recycle], includes = [null]
14:41:53,321 DEBUG [f.p.e.c.f.f.FsCrawlerUtil] filename = [#recycle], excludes = [[#recyle]]
14:41:53,321 DEBUG [f.p.e.c.f.FsParserAbstract] [#recycle] can be indexed: [true]
14:41:53,321 DEBUG [f.p.e.c.f.FsParserAbstract]   - folder: #recycle

Pour ce qui est de mon yaml  il est basique: 

```yml
update_rate: "15m"
  excludes:
  - "#recyle"
  json_support: false
```

J'ai essaye des variantes en mettant,  `- "*#recyle*"`, ou  `- "\\#recyle"`

Avez-vous des idées du problème, s'il vous plaît?

**Comment by dadoonet** [comment]
Bonjour


Je me demande si ça ne devrait pas être `- "\#recyle"`?
